python/xorbits/numpy/__init__.py

```diff
@@ -74,8 +74,10 @@
 
 def _install():
     from .mars_adapters import _install as _install_mars_adapters
```

Comment:
```suggestion
    from .mars_adapters import _install as _install_mars_methods
```

To make it more consistent.